content/en/monitors/types/process.md

```diff
@@ -19,21 +19,44 @@ further_reading:
 
 ## Overview
 
-Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+Live Process Monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+
+Live Process Monitors are recommended to:
+
+- Ensure that enough instances of a non-containerized long-lived process are running.
+- Flag when a specific process is running.
+
+**Note**: Only long-lived processes are picked up by the Agent. Monitors on processes that live for less than 20 seconds may be flaky.
 
 ## Monitor creation
 
-To create a [live process monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
+To create a [Live Process Monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
 
 ### Select processes
 
-Search for a process to monitor using a space separated list of strings. The search performs a partial match, fuzzy search across all processes on your infrastructure. Search operators `AND`, `OR`, and `NOT` are supported. See the [Live Process Monitoring][3] page for details. Additionally, filter by tags to refine your monitor to a specific scope.
-
-Matching processes and counts are displayed below the search:
+You can use either tags or a fuzzy search to filter across all processes in your infrastructure. Matching processes and counts are displayed below the search:
```

Comment:
Should we avoid using "fuzzy" here? @robertjli @wiyu @kkhor-datadog @badovm @mbotarro for thoughts

Reply:
We already mention fuzzy search in a couple of places so it seems fine, but seems worth clarifying this is for the text search only i.e. "fuzzy search" -> "fuzzy text search"

https://github.com/search?q=repo%3ADataDog%2Fdocumentation%20fuzzy&type=code

Reply:
Updated!